### PR TITLE
Updates to ML Use Case

### PR DESCRIPTION
**Pull request by jpvantassel**
# In this PR

- ✨ Add description and overview of use case.
- ✨ Add links to ml notebooks in community data.
- 🔥 Remove unused image file.
- 🚚 Change `ml_resources` to `vantassel_and_zhang`.

### CODE REVIEW

File: docs/vantassel_and_zhang/usecase.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+an image of a hand-written digit and predict a label classifying it as a number between 0 and 9. The notebooks
+will show how to install Keras/TensorFlow, load a standard dataset, pre-process the data for acceptance by the
+network, design and train a convolutional neural network using Keras/TensorFlow, and visualize output correct and
+incorrect output predictions. For those who have access to graphical processing unit (GPU) computational
+resources a replica of the main notebook is provided that can run across multiple GPUs on a single system.
+
+[Convolutional Neural Network Example](https://www.designsafe-ci.org/data/browser/public/designsafe.storage.community/Use%20Case%20Products/An%20Example-Based%20Introduction%20to%20Machine%20Learning/3_convolutional_neural_networks)
+
+
+## Citation and Licensing
```
Comment (kks32) [inline docs/vantassel_and_zhang/usecase.md]: This needs to be at the top

**Review by kks32 — CHANGES_REQUESTED** [review]
Rearrange to move citations to the top of the page

File: docs/vantassel_and_zhang/usecase.md (hunk 1 above)
Comment (jpvantassel) [reply docs/vantassel_and_zhang/usecase.md]: Done

File: docs/vantassel_and_zhang/usecase.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+<!-- * Please cite [AUTHORS et al. (20xx) - example of published project]() to acknowledge the use of any resources from this use case. -->
+
+* Please cite [Rathje et al. (2017)](https://doi.org/10.1061/(ASCE)NH.1527-6996.0000246) to acknowledge the use of DesignSafe resources.  
+
+* Please cite [Durante and Rathje (2021)](https://doi.org/10.1177/87552930211004613) to acknowledge the use of any resources for the Random Forest and Neural Networks examples included in this use case.
+
+* This software is distributed under the GNU General Public License ([https://www.gnu.org/licenses/gpl-3.0.html](https://www.gnu.org/licenses/gpl-3.0.html)).
```
Comment (kks32) [inline docs/vantassel_and_zhang/usecase.md]:
```suggestion
* This software is distributed under the [GNU General Public License](https://www.gnu.org/licenses/gpl-3.0.html).
```

**Review by kks32 — CHANGES_REQUESTED** [review]
Can you give me edit access?

- Fix mkdocs.yml file folder `ml_resources`
- GPL license link change

**Review by kks32 — APPROVED** [review]

**Pull request merged**